### PR TITLE
adding gitleaks precommit config

### PR DESCRIPTION
**Pull request by gfreeman-navapbc**
## 🎫 Ticket

https://jira.cms.gov/browse/PLT-3 Add pre-commit config for gitleaks to all repos

## 🛠 Changes
Add pre-commit config for gitleaks to all repos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,12 +77,36 @@ Some variables below have a note indicating their name should be changed. These
...
+```sh
+brew install pre-commit
+```
+
+Other installation options can be found in the [pre-commit documentation](https://pre-commit.com/#install).
+
 To avoid committing and pushing unencrypted secret files, use the included `scripts/pre-commit` git pre-commit hook by running the following script from the repository root directory:
 
 ```
 cp ops/pre-commit .git/hooks
```
Comment (gsf) [inline README.md:97]: Another chance to drop the ops/pre-commit in favor of pre-commit tool.

**Review by gsf — DISMISSED** [review]
Looks good though https://github.com/CMSgov/bcda-app/pull/839#discussion_r1194113733 applies here as well about adding go-imports to `.pre-commit-config.yaml`.

**Review by kyeah — DISMISSED** [review]
Confirmed locally, thanks!

Similar to bcda-app, can we delete the old scripts/pre-commit file? thanks!

**Review by gsf — APPROVED** [review]
lgtm though you may want to get a final confirmation from @kyeah

**Review by kyeah — APPROVED** [review]
Looks good (I've been meaning to recommend some git repo setting changes like removing "dismiss stale approvals", but that's for a later day..)

**Pull request merged**